components/rating.css

```diff
@@ -0,0 +1,14 @@
+@import url(/stylesheets/main.css);
+
+.ratings-container{
+    width: 30rem;
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    justify-content: center;
+    color: rgb(255,164,29);
+}
```

Comment:
If u are using the variables for colors then you might wanna change this to a variable too for the sake of consistency

Reply:
changed the color name to appropriate naming conventions in latest commits.

Reply:
made necessary changes in latest commits.